tests/functional/c/consider/consider_using_with.py

```diff
@@ -2,13 +2,24 @@
 import codecs
 import contextlib
 import multiprocessing
+import pathlib
 import subprocess
 import tarfile
 import tempfile
 import threading
 import urllib
 import zipfile
 from concurrent.futures import ThreadPoolExecutor, ProcessPoolExecutor
+from pathlib import Path
+
+
+def test_pathlib_open():
+    _ = pathlib.Path("foo").open(encoding="utf8")  # [consider-using-with]
+    _ = Path("foo").open(encoding="utf8")  # [consider-using-with]
+    path = Path("foo")
+    _ = path.open(encoding="utf8")  # [consider-using-with]
+    with Path("foo").open(encoding="utf8") as file:  # must not trigger
```

Comment:
` # must not trigger` comment not really necessary

Reply:
I saw it in other tests, so I thought I should add it too. Should I remove it?

Reply:
I vote yes but it's also not a big deal at all.

Reply:
Yeah it's redundant, probably better to keep it for where it's ambiguous if it should raise or if it's an expected false negative but also let's not rerun a CI for that :)